carma-messenger-core/truck_inspection_plugin/src/truck_inspection_plugin.cpp

```diff
@@ -51,6 +51,34 @@ namespace truck_inspection_plugin
     {
         cav_msgs::MobilityRequest msg;
         msg.strategy = TruckInspectionPlugin::INSPECTION_STRATEGY;
+        //add header temporarily for ACE testing, will be removed once the mobilityRequest message node is fixed
```

Comment:
Per Discussion on Friday, the fix https://github.com/usdot-fhwa-stol/carma-messenger/pull/45 should resolve this issue and your changes on this file can be reverted.

Reply:
Reverted